src/librbd/cache/pwl/rwl/WriteLog.cc

```diff
@@ -888,7 +889,7 @@ void WriteLog<I>::reserve_cache(C_BlockIORequestT *req,
     buffer.allocation_lat = ceph_clock_now() - before_reserve;
     if (TOID_IS_NULL(buffer.buffer_oid)) {
       if (!req->has_io_waited_for_buffers()) {
-        req->set_io_waited_for_entries(true);
```

Comment:
I think  no need check "req->has_io_waited_for_buffers".  If it true, we can't do this.